### DISCUSSION TITLE
Pinia // Firebase // VueFire

### DISCUSSION THREAD

**Discussion by websiddu** [vuejs/vuefire#1276] [Questions and Help] [answered]
Is it possible to have an adapter of some bindings that will combine Pinia with VueFire, I really liked VuexFire, and worked like magic and extremely fast? 

Are there any plans for such a thing, if not how do I achieve something like that?

**Comment by posva** [comment, marked as answer]
https://vuefire.vuejs.org/cookbook/subscriptions-external.html#pinia

**Reply by websiddu** [reply]
Ahh.. I should have checked the docs, before posting this. Thanks I will try it out.
